### PR TITLE
Tailsitter auto transition 

### PR DESCRIPTION
**Pull request by davidvor**
test flights:
http://logs.uaventure.com/view/M83mXsytSLkGty6XhLqwpj
http://logs.uaventure.com/view/DizsjJkFyfMwBKrkQFjCPY

front transition  MC->FW works both in manual and altctrl modes
back transition FW->MC works only in manual mode

### CODE REVIEW

File: src/modules/vtol_att_control/tailsitter.cpp [hunk 1]
```diff
@@ -55,22 +81,153 @@ Tailsitter::~Tailsitter()
 
 }
 
+int
+Taulsitter::parameters_update()
```
Comment (SimonWilks) [inline src/modules/vtol_att_control/tailsitter.cpp]: Typing error.

**Comment by SimonWilks** [comment]
@davidvor you may have already realised but just in case could you rebase on master? This is pulling in a lot of old changes.

**Comment by davidvor** [comment]
@SimonWilks  i don't mean to merge the old changes into master, it only describes the technique for auto transition for now, rebuilding on master  will be done after some base changes in master will allow virual set points and right orientation set points in FW mode

**Comment by davidvor** [comment]
wrong branch :/

**Pull request closed without merging**